tools/taskcluster/build_targets.py

```diff
@@ -33,14 +33,14 @@
 import sys
 
 platforms = {
-   'oculusvr': ['arm', 'arm64'],
-   'oculusvrStore': ['arm', 'arm64'],
-   'oculusvr3dofStore': ['arm', 'arm64'],
-   'wavevr': ['arm', 'arm64'],
+   'oculusvr': ['arm64'],
+   'oculusvrStore': ['arm64'],
+   'oculusvr3dofStore': ['arm64'],
+   'wavevr': ['arm64'],
    'wavevrStore': ['arm', 'arm64'],
```

Comment:
Do we still wanna support `arm` as the first option for wavevrStore? `return [archList[0]]` at Ln 66 will make `arm` to be the first choice.

Reply:
https://github.com/MozillaReality/FirefoxReality/pull/1867/files#diff-0d8ef6f62b99db4845f293a1ada2cd85R59
Makes `arm64` the default if nothing is specified..